src/ServiceBuilder.php

```diff
@@ -92,7 +92,7 @@ public function __construct(array $config = [])
      * ```
      * use Google\Cloud\ServiceBuilder;
      *
-     * // Create a storage client using application default credentials.
+     * // Create a Storage client using application default credentials.
```

Comment:
Oh, interesting. That could work. I can make that change if it reads better to you.